apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationInformation/ApplicationInformation.tsx

```diff
@@ -234,13 +220,13 @@ const ApplicationInformation = ({
             <PersonalInformationDisplay user={snapshot} />
           </Accordion.Content>
         </Accordion.Item>
-        {mergedQuestionResponses.length > 0 ? (
+        {screeningQuestionResponses.length > 0 ? (
```

Comment:
It looks like we display a different component if the flag is off - are you viewing this in storybook?

Reply:
Yeah, there's a totally different page with the flag off.  The workaround you removed is so that this component can render snapshots where general questions are still called screening questions.  Vachan recently added this in, in https://github.com/GCTC-NTGC/gc-digital-talent/issues/9415.

Reply:
Ok, I've merged main. That should have brought the fix in.

Reply:
It works!  :clap: